### PR TITLE
Plotly quantile support

### PR DESCRIPTION
**Pull request by karl-richter**
Added support for the plotting of uncertainty quantiles to plotly.
Tested with the feature-use tutorial notebooks created by @Kevin-Chen0.
Added plotting test for the plotly uncertainty plotting functionality.

Building on the ideas by #368, but created a new PR due to incompatible changes.

### CODE REVIEW

**Comment by Kevin-Chen0** [comment]
When running this code, should the plot also be shown?

`fig_forec = m.plot(forecast, plotting_backend="plotly")`

Just like for `pyplot` as shown below.

plt_plot

**Comment by karl-richter** [comment]
> When running this code, should the plot also be shown?
> 
> 
> 
> `fig_forec = m.plot(forecast, plotting_backend="plotly")`
> 
> 
> 
> Just like for `pyplot` as shown below.
> 
> 
> 
> plt_plot

Hi Kevin,
currently no. Currently you always have to call .show() on the object to display the graph. I can check whether we can change that behavior, but it should not be something uncertainty related but in general for all plotly plots. Thanks for the comment!

**Comment by Kevin-Chen0** [comment]
Got it, then for `pyplot`, should the plotting also be suppressed when just doing an assignment? If so, then can we also add that change as part of this PR? @ourownstory

`fig_forec = m.plot(forecast)`

**Comment by karl-richter** [comment]
@ourownstory I performed the changes as requested yesterday, are we good to go?

File: neuralprophet/forecaster.py [hunk 1]
```diff
@@ -1451,17 +1451,14 @@ def plot(self, fcst, df_name=None, ax=None, xlabel="ds", ylabel="y", figsize=(10
         )
         if plotting_backend == "plotly":
             log.info("Plotly does not support plotting of quantiles yet.")
```
Comment (ourownstory) [inline neuralprophet/forecaster.py:1453]: don't forget to remove these logs

Comment (Kevin-Chen0) [reply neuralprophet/forecaster.py:1453]: I'll remove those within an hour or 2.

Comment (Kevin-Chen0) [reply neuralprophet/forecaster.py]: done

**Review by Kevin-Chen0 — APPROVED** [review]
Looks good!

**Pull request merged**